src/renderer/component/categoryList/view.jsx

```diff
@@ -1,12 +1,13 @@
 // @flow
```

Comment:
I didn't do much here, mostly switched to `createRef` and made it so it played nice with the new scrollable wrapper.